[#55] Add showflags command

MithicSpirit opened the pull request:
Seems like an oversight to be able to hide flags but not show them again, so this attempts to fix that. I am unable to test it but I believe that what I have so far should work, though it may also be desirable to update the documentation and add its own permission (rather than just using `sr.hide`).

MithicSpirit committed:
Add showflags command

Nearly identical in every aspect to hideflags, except that it makes the
flags appear again.

galaipa commented:
Hello,
Thank you very much for taking the time to implement the feature. I just tried it and it works like a charm. 

I suggest adding a new permission called "sr.flag" that would allow both /sr hideflags and showflags. What do you think? I'm open to other ideas. Once we decided this I guess I could merge your pull request to master and publish a new release.

Regarding testing the changes, if the problem is technical, it should be pretty easy. You just need to run `mvn package` in the root of the repo, and the jar will be created in the `target` folder. Please let me know if you have any problem with this. 

Thank you again

MithicSpirit commented:
For the permissions, I think that it would be better to have one for hideflags (`sr.hide`) and one for showflags (`sr.show` maybe?) to allow for more granular permission control. 
As for testing my primary issue was that I didn't have a server I could run it on haha, but now I set one up locally and it's working fine.

galaipa commented:
Fair enough! `sr.show` sounds good. Please, could you add the permission and update the pull request? I'm short in time lately to do it myself.
Thank you

MithicSpirit committed:
sr.show permission for showflags

Creates a new sr.show permission and assigns /sr showflags to use it
instead of sr.hide (which is used for /sr hideflags).

MithicSpirit commented:
Ok, that last commit added `sr.show`. I also tested it on my local server with luckperms and seems to be working correctly; I believe that this is now ready to be merged.

galaipa commented:
Great! I will merge and publish a new release.

Thank you for your work!
